### PR TITLE
Tracy\Panel: added vector icon

### PR DESCRIPTION
**Pull request by bckp**
I added SVG icon from Nette\Database 2.3.0b3

### CODE REVIEW

**Comment by dg** [comment]
:+1:

File: dibi/bridges/Tracy/Panel.php [hunk 1]
```diff
@@ -81,13 +81,14 @@ public static function renderException($e)
...
-			. count($this->events) . ' queries'
+		return '<span title="dibi"><svg viewBox="0 0 2048 2048"><path fill="' . ( $count ? '#b079d6' : '#aaa') . '" d="M1024 896q237 0 443-43t325-127v170q0 69-103 128t-280 93.5-385 34.5-385-34.5-280-93.5-103-128v-170q119 84 325 127t443 43zm0 768q237 0 443-43t325-127v170q0 69-103 128t-280 93.5-385 34.5-385-34.5-280-93.5-103-128v-170q119 84 325 127t443 43zm0-384q237 0 443-43t325-127v170q0 69-103 128t-280 93.5-385 34.5-385-34.5-280-93.5-103-128v-170q119 84 325 127t443 43zm0-1152q208 0 385 34.5t280 93.5 103 128v128q0 69-103 128t-280 93.5-385 34.5-385-34.5-280-93.5-103-128v-128q0-69 103-128t280-93.5 385-34.5z"/></svg><span class="tracy-label">'
```
Comment (dg) [inline dibi/bridges/Tracy/Panel.php]: Please add here attribute `style="vertical-align: bottom; width:1.23em; height:1.55em"`

**Comment by janlanger** [comment]
:100: I just came here to prepare similar pull :smiley:

File: dibi/bridges/Tracy/Panel.php (hunk 1 above)
Comment (bckp) [inline dibi/bridges/Tracy/Panel.php]: Into <span title="dibi"> or into <svg> tag?

Comment (dg) [inline dibi/bridges/Tracy/Panel.php]: title in span, style in svg

**Comment by bckp** [comment]
@dg ping :)

**Comment by dg** [comment]
Can you squash it to single commit?

**Comment by bckp** [comment]
@dg how? Only by new PR, or is there other way?

**Comment by dg** [comment]
http://eli.thegreenplace.net/2014/02/19/squashing-github-pull-requests-into-a-single-commit

**Comment by bckp** [comment]
@dg OK, I will install git and tomorow will squash it.

**Comment by dg** [comment]
:+1:

**Comment by bckp** [comment]
@dg Done :) GitHub for windows have Git shell :)

**Comment by dg** [comment]
last thing: please change commit message to "Tracy\Panel: added vector icon"

**Comment by bckp** [comment]
@dg fixed :) have learn lot, thanks

**Comment by dg** [comment]
Thank you!

**Pull request merged**